DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -90,12 +97,33 @@ user = postfix
 password = password
 hosts = localhost
 dbname = postfix
-query          = SELECT domain FROM domain WHERE domain='%s' AND active = '1'
+query          = SELECT domain FROM domain WHERE domain='%s' AND active = '1' AND ( primarymx='$myhostname' OR primarymx='' )
 #query          = SELECT domain FROM domain WHERE domain='%s'
 #optional query to use when relaying for backup MX
 #query           = SELECT domain FROM domain WHERE domain='%s' AND backupmx = '0' AND active = '1'
 #expansion_limit = 100
 
+mysql_relay_domains_maps.cf:
+user = postfix
+password = password
+hosts = localhost
+dbname = postfix
+query = SELECT domain FROM domain WHERE domain='%s' AND active = '1' AND primarymx !='$myhostname' AND primarymx!=''
+
+mysql_backup_mx_recipients.cf
```

Comment:
minor nitpick: the ':' at the end of the line is missing (which will break the scripting part of POSTFIX_CONF.TXT)